oak_containers_hello_world_untrusted_app/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use oak_containers_launcher::Launcher;
+
+mod app_client;
+
+/// The local address that will be forwarded by the VMM to the guest on port 8080.
+const UNTRUSTED_APP_ADDRESS: &str = "http://127.0.0.1:8088";
```

Comment:
Ideally the local host port would be specified by the caller (or auto assigned -- though auto assignment is problematic because the caller needs to know it, so let's leave that aside for now).
Can we pass it via the launcher args?